lib/FieldType/Mapper/UserFormMapper.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\FieldType\Mapper;
+
+use EzSystems\RepositoryForms\Data\FieldDefinitionData;
+use EzSystems\RepositoryForms\Form\Type\UserType;
+use Symfony\Component\Form\FormInterface;
+
+class UserFormMapper extends AbstractMapper
```

Comment:
Not sure we should have this one at all. User isn't editable via the Content API...

We also can't really use it for registration, since the `UserCreateStruct` doesn't use fields for user account data (it has first level properties for username, email, password...).